lms/envs/aws.py

```diff
@@ -795,3 +795,6 @@
 # The extended StudentModule history table
 if FEATURES.get('ENABLE_CSMH_EXTENDED'):
     INSTALLED_APPS += ('coursewarehistoryextended',)
+
+if ENV_TOKENS.get('API_ACCESS_MANAGER_EMAIL', None):
```

Comment:
Done. I actually kind of like the pattern of setting an example value in the base settings, then overriding it with either a real value or `None`. That provides some code-as-documentation while also squawking loudly when things aren't set to real values.